core/server/middleware/index.js

```diff
@@ -79,14 +78,17 @@ setupMiddleware = function setupMiddleware(blogApp) {
     // (X-Forwarded-Proto header will be checked, if present)
     blogApp.enable('trust proxy');
 
-    // Logging configuration
-    if (logging !== false) {
-        if (blogApp.get('env') !== 'development') {
-            blogApp.use(logger('combined', logging));
-        } else {
-            blogApp.use(logger('dev', logging));
-        }
-    }
+    /**
+     * request logging
+     * @TODO: add error to output
+     */
+    blogApp.use(function expressLogging(req, res, next) {
+        res.once('finish', function () {
+            logging.request({req: req, res: res});
+        });
```

Comment:
In the #7116 issue the output form morgan looks like:

```
GET / 200 140.121 ms - -
GET /assets/js/jquery.fitvids.js?v=4d853bd599 200 17.077 ms - -
GET /assets/js/index.js?v=4d853bd599 200 16.385 ms - -
GET /assets/css/screen.css?v=4d853bd599 200 19.621 ms - -
```

This includes a duration, which is missing from the current code. Again, this can be a TODO for a later iteration, however I think we should use the same code for logging to express as morgan does.